src/components/TrialBanner.js

```diff
@@ -0,0 +1,173 @@
+import _ from 'lodash/fp'
+import { div, h, a, span } from 'react-hyperscript-helpers'
+import { authStore, refreshTerraProfile } from 'src/libs/auth'
+import colors from 'src/libs/colors'
+import { buttonPrimary, Clickable, LabeledCheckbox } from 'src/components/common'
+import { icon, spinner } from 'src/components/icons'
+import { ajaxCaller } from 'src/libs/ajax'
+import { reportError } from 'src/libs/error'
+import * as Utils from 'src/libs/utils'
+import { Component } from 'src/libs/wrapped-components'
+import Modal from 'src/components/Modal'
+import FreeTrialEulas from 'src/components/FreeTrialEulas'
+
+
+const messages =
+  {
+    'Enabled': {
+      'title': 'Welcome to Terra!',
+      'message': 'You have free compute and storage credits available to upload your data and launch analyses.',
+      'isWarning': false,
+      'enabledLink': {
+        'label': 'Learn more',
+        'url': 'https://software.broadinstitute.org/firecloud/documentation/freecredits'
+      },
+      'button': {
+        'label': 'Start trial',
+        'isExternal': false
+      }
+    },
+    'Enrolled': {
+      'title': 'Access Free Credits',
+      'message': 'You currently have access to your free credits. Learn how to use FireCloud, about this free credit period, and transitioning to your own billing account once the free credits have expired.',
+      'isWarning': false,
+      'button': {
+        'label': 'Learn More',
+        'url': 'https://software.broadinstitute.org/firecloud/documentation/freecredits',
+        'isExternal': true
+      }
+    },
+    'Terminated': {
+      'title': 'Your free credits have expired',
+      'message': 'Your data will be stored for 30 days from credit expiration date. Learn how you can create your own Google Billing Account and move your data to continue working.',
+      'isWarning': true,
+      'button': {
+        'label': 'Learn more',
+        'url': 'https://software.broadinstitute.org/firecloud/documentation/freecredits?page=faqs',
+        'isExternal': true
+      }
+    }
+  }
+
+export const TrialBanner = _.flow(
+  ajaxCaller,
+  Utils.connectAtom(authStore, 'authState')
+)(class TrialBanner extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      accessingCredits: false,
+      pageTwo: false,
+      termsAgreed: 'false',
```

Comment:
Is there a reason for these to be strings instead of boolean values?

Reply:
I'm not totally sure why its the case, but I modeled it off how we do the Labeled checkboxes in Profile. Converting it to regular boolean breaks it somehow, and I think it has something to do with the onChange and .toString() method. I wasn't really sure how to do it in any other way, so I just adopted what I knew worked from the Profile!